1.Practicals_C/Decision_Making/Looping/Implementation/sum.c

```diff
@@ -0,0 +1,17 @@
+#include <stdio.h>
```

Comment:
Write in one line or two that what's the problem statement this program is intending to solve.

Reply:
ok sir !!